distributed/nanny.py

```diff
@@ -356,14 +356,17 @@ def start(self):
                 yield self.process.start()
                 if self.status == 'starting':
                     yield gen.with_timeout(timedelta(seconds=5),
-                                           self._wait_until_running())
+                                           self._wait_until_started())
             except gen.TimeoutError:
                 logger.info("Failed to start worker process.  Restarting")
                 yield gen.with_timeout(timedelta(seconds=1),
```

Comment:
Not sure if it is needed, but it might be nice if this was configurable too.

Reply:
Yeah, ideally all of the timeouts everywhere would be configurable.  I'm not yet sure how best to do this.